src/components/application_manager/include/application_manager/core_service.h

```diff
@@ -71,27 +71,25 @@ class CoreService : public Service {
   /**
    * Checks access to requested equipment of vehicle
    * @param app_id id of application
-   * @param function_id name of RPC
+   * @param module type
    * @param params parameters list
    * @param seat seat of owner's mobile device
    * @return return allowed if access exist,
    * manual if need to send question to driver otherwise disallowed
    */
   virtual TypeAccess CheckAccess(const ApplicationId& app_id,
```

Comment:
CheckPermissions and CheckAccess are different features. CheckAccess is used only for the checking reverse functionality. CheckPermissions is invoked before CheckAccess. Did I correctly understand your question?